my-react/src/App.jsx

```diff
@@ -1,35 +1,86 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
+import { useLocalStorage } from "./use_local_storage.jsx";
+import { List } from "./List.jsx";
+import { Add } from "./Add.jsx";
+import { Edit } from "./Edit.jsx";
+import "./App.css";
 
-function App() {
-  const [count, setCount] = useState(0)
+const initial_data = [
+  { id: 0, content: "松尾芭蕉\n古池や蛙飛び込む水の音" },
+  { id: 1, content: "小林一茶\n痩せガエル負けるな一茶ここにあり" },
+  { id: 2, content: "作者不詳\n富士山麓に鸚鵡鳴く" },
+];
+
+export default function App() {
+  const [draft, setDraft] = useState("");
+  const [selectedId, setSelectedId] = useState(null);
+  const [status, setStatus] = useState("list");
+  const [notes, setNotes] = useLocalStorage(initial_data);
+
+  const ids_titles = getIdsTitles();
+
+  function getIdsTitles() {
+    return notes.map((data) => {
+      const ary = data.content.split("\n");
```

Comment:
```suggestion
      const array = data.content.split("\n");
```

変数名はなるべく誰がみてもわかるものを！

Reply:
arrayに変更しました！